src/modem/OwlModemNetwork.cpp

```diff
@@ -517,3 +577,61 @@ int OwlModemNetwork::getSignalQuality(at_csq_rssi_e *out_rssi, at_csq_qual_e *ou
   }
   return 1;
 }
+
+int OwlModemNetwork::setEDRXMode(at_edrx_mode_e n,
+    at_edrx_access_technology_e t,
+    at_edrx_cycle_length_e v) {
+  if (t == AT_EDRX_Access_Technology__Unspecified) {
+    atModem_->commandSprintf("AT+CEDRXS=%d", n);
+  } else {
+    if ((n == AT_EDRX_Mode__Enabled || n == AT_EDRX_Mode__Enabled_With_URC) &&
+        (v != AT_EDRX_Cycle_Length__Unspecified)) {
+      char v_str_[8];
+      str v_str = { .s = v_str_, .len = 0 };
+      uint8_t_to_binary_str(v, &v_str, 4);
+      atModem_->commandSprintf("AT+CEDRXS=%d,%d,\"%.*s\"", n, t, v_str.len, v_str.s);
+    } else {
+      atModem_->commandSprintf("AT+CEDRXS=%d,%d", n, t);
+    }
+  }
+
+  return atModem_->doCommandBlocking(180 * 1000, nullptr) == AT_Result_Code__OK;
```

Comment:
Not related to this PR: I really want us to get rid of `doCommandBlocking` with arbitrary timeouts at some point, that is not really manageable to me.

Reply:
A good improvement - lets get a card added for this to the backlog